textpipe/doc.py

```diff
@@ -10,7 +10,10 @@
 import textacy
 import textacy.text_utils
 from bs4 import BeautifulSoup
-
+from pattern.text.nl import sentiment as sentiment_nl
+from pattern.text.en import sentiment as sentiment_en
+from pattern.text.fr import sentiment as sentiment_fr
+from pattern.text.it import sentiment as sentiment_it
```

Comment:
Now we load all languages in memory, regardless of which you'll use/need. 
Not sure how to make this more lazy, but that would be nice. Any thoughts on it?

On second look, it does look like it is loaded lazily; 

```
In [1]: from pattern.text.en import sentiment as sentiment_en

In [2]: import sys

In [3]: sys.getsizeof(sentiment_en)
Out[3]: 256
[...] -> print(sentiment_en)
In [5]: sys.getsizeof(sentiment_en)
Out[5]: 147576
```

Reply:
It takes ~4sec to load the EN object;
```
In [3]: t0=time();from pattern.text.en import sentiment as sentiment_en;print(time()-t0)
3.775541067123413
```
The other languages come for free;
```
In [4]: t0=time();from pattern.text.en import sentiment as sentiment_fr;print(time()-t0)
6.103515625e-05

In [5]: t0=time();from pattern.text.en import sentiment as sentiment_nl;print(time()-t0)
6.198883056640625e-05

In [6]: t0=time();from pattern.text.en import sentiment as sentiment_de;print(time()-t0)
7.104873657226562e-05
```